runtime/appruntime/api/reqtrack.go

```diff
@@ -97,6 +97,11 @@ func (s *Server) beginRequest(ctx context.Context, p *beginRequestParams) (*mode
 		curr.Trace.BeginRequest(req, curr.Goctr)
 	}
 
+	s.customMetrics.Counter("e-requests-total", map[string]string{
```

Comment:
I prefixed the counter name with `e` to make it less likely for a user to pick the same metric name (when we support user-defined metrics).